cmd/juju/application/addrelation.go

```diff
@@ -117,6 +117,11 @@ func (c *addRelationCommand) Run(ctx *cmd.Context) error {
 	if params.IsCodeUnauthorized(err) {
 		common.PermissionsMessage(ctx.Stderr, "add a relation")
 	}
+	if params.IsCodeAlreadyExists(err) {
+		// It's not a real error, log it and move along
+		logger.Infof("%s", err)
```

Comment:
We really should print something to the console. logging like this only shows up with --show-log
Something like:
$ juju relate ubuntu telegraf
relation between ubuntu and telegraf already exists

The above is done using ctx.Infof()